monkey/infection_monkey/Pipfile

```diff
@@ -5,8 +5,7 @@ name = "pypi"
 
 [packages]
 cryptography = "==2.5" # We can't build 32bit ubuntu12 binary with newer versions of cryptography
-pyinstaller = {git = "git://github.com/guardicore/pyinstaller"}
-pyinstaller-hooks-contrib = "==2021.1" # Required to build docker with our pyinstaller branch
+pyinstaller = "==4.2"
```

Comment:
I think we can also drop `altgraph` and `importlib-metadata`. I believe we only needed them due to import issues with our fork. Specifying upstream pyinstaller should include those libraries. I would suggest removing them from the `Pipfile`, running `pipenv lock`, and checking the `Pipfile.lock` to ensure they're still there.